spec/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager/cloud_manage_spec.rb

```diff
@@ -0,0 +1,40 @@
+describe ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager do
+  context ".raw_connect" do
+    let(:config) do
+      {
+        :provider               => "IbmCloud",
+        :IbmCloud_project         => "project",
+        :IbmCloud_json_key_string => "encrypted",
+        :app_name               => Vmdb::Appliance.PRODUCT_NAME,
+        :app_version            => Vmdb::Appliance.VERSION,
+        :Ibm_client_options  => { :proxy_url => "proxy_uri" },
+      }
+    end
+  end
+  it ".ems_type" do
+    expect(described_class.ems_type).to eq('ibm_cloud_power_virtual_servers')
+  end
+
+  it ".description" do
+    expect(described_class.description).to eq('IBM Cloud Power Systems Virtual Server')
```

Comment:
Looks like this is failing on a valid bug, the description [here](https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/power_virtual_servers/cloud_manager.rb#L58) should be changed to `"IBM Cloud Power Systems Virtual Servers"` (should be plural though)

Reply:
I submitted #9 to address this. The official product description is `"IBM Power Systems Virtual Servers"` (without 'Cloud'). @bashathadikela once that get finalized and merged you'll need to update this test to reflect the new name and rebase.